planner/core/expression_rewriter.go

```diff
@@ -1814,6 +1824,36 @@ func (er *expressionRewriter) evalDefaultExpr(v *ast.DefaultExpr) {
 	er.ctxStackAppend(val, types.EmptyName)
 }
 
+// GetVar try fold scope start
```

Comment:
The comment header is better to start with the Function Name.
`getVarTryFoldOrNotStart does somehing ....`

And it would be better if we explain what does `OrNotStart` mean.

Reply:
Ok, I will fix this.